lib/profile/sections/skills.dart

```diff
@@ -1,103 +1,88 @@
 import 'package:flutter/material.dart';
 import 'package:google_fonts/google_fonts.dart';
 import 'package:mentorApp/constants/profile_constants.dart';
-import 'package:percent_indicator/percent_indicator.dart';
-import './header.dart';
+import './section.dart';
  
 class ProfileSkills extends StatelessWidget {  
   final List<Map<String, Object>> skills;
   ProfileSkills({Key key, @required this.skills}) : super(key: key);
   
   @override
   Widget build(BuildContext context) {
-    return Container(
-      width: MediaQuery.of(context).size.width,
-      padding: EdgeInsets.fromLTRB(30.0, 10.0, 30.0, 20.0),
-      decoration: BoxDecoration(
-        border: Border(
-          bottom: BorderSide(
-            width: 5.0,
-            color: Colors.black12
-          )
-        )
-      ),
-      child: Column(
-        crossAxisAlignment: CrossAxisAlignment.start,
-        children: <Widget>[
-          SectionHeader(title: sectionTitleSkills),
-          buildSkillList(context)
-        ],
-      )
+    return Section(
+      title: sectionTitleSkills,
+      child: buildSkillList(context)
     );
   }
 
-  buildSkillRow(BuildContext context, String skill, double rating) {
-    return Container(
-      width: MediaQuery.of(context).size.width,
-      padding: EdgeInsets.fromLTRB(0.0, 10.0, 0.0, 10.0),
-      decoration: BoxDecoration(
-        border: Border(
-          bottom: BorderSide(
-            color: Colors.black12
-          )
-        )
-      ),
-      child: Column(
-        crossAxisAlignment: CrossAxisAlignment.start,
-        children: <Widget>[
-          buildSkill(skill),
-          buildRatingBar(context, rating)
-        ]
-      )
+  buildSkillRow(BuildContext context, String skill, String level) {
+    return SectionRow(
+      children: <Widget>[
+        buildSkillText(skill, true),
+        buildSkillText(level, false)
+      ]
     );
   }
 
   buildSkillList(BuildContext context) {
-    List<Widget> skillList = new List<Widget>();
+    List<Widget> expertList = new List<Widget>();
+    List<Widget> intermediateList = new List<Widget>();
+    List<Widget> beginnerList = new List<Widget>();
+
     for (int i = 0; i < skills.length; i++) {
-      skillList.add(
-        buildSkillRow(
-          context, 
-          skills[i]["skill"],
-          skills[i]["rating"]
-        )
-      );
+      if (skills[i]["level"] == skillExpert){
```

Comment:
Not sure how to optimize this right now. Thinking of some type of switch function that would decide the parameters for `buildSkillRow`. This can be left like this for now though I think.

Reply:
I think later on once this is hooked to the backend, our query could order these by skillLevel and potentially the UI would not have to do any sorting?

Reply:
yep I was thinking the same thing!

Reply:
NICE